raclambda/raclambda/raclambda_stack.py

```diff
@@ -26,6 +26,7 @@ def __init__(
         project_name: str,
         queue_arn_export_name: str,
         lambda_timeout: Duration = Duration.seconds(300),
+        dregs_expiration: Duration = Duration.days(7),
```

Comment:
This means that files will only be kept for a week, right?
Is this the mechanism that cleans up used dregs that are no longer useful?

Reply:
Yes, that is the idea.